lib/default-theme/NavLink.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+  <router-link
+    class="router-link"
+    :to="item.link"
+    v-if="!isExternal(item.link)">
+    {{ item.text }}
+  </router-link>
+  <a
+    v-else
+    :href="item.link"
+    target="_blank"
+    class="router-link">
+    {{ item.text }}
+  </a>
+</template>
+
+<script>
+  import { isExternal } from './util'
+  export default {
+    props: {
+      item: {
+        required: true
+      }
+    },
+    mixins: [
```

Comment:
why do you use a mixin here?

Reply:
thanks, using a mixin here is to reuse the existing method. or you would have a better way to do it.

Reply:
you can just put the methods object without the mixins, it would also work

Reply:
Cool, long time to write Vue, I just forgot some important thing, thanks.